packages/components/package.json

```diff
@@ -76,8 +76,8 @@
     "uuid": "^8.3.2"
   },
   "peerDependencies": {
-    "react": ">=16.8",
-    "react-dom": ">=16.8"
+    "react": "^18",
+    "react-dom": "^18"
```

Comment:
Why is this needed? Should we not let consumers of this (ie non-jobber folk) decide on the react version? Some people do not check the dependencies for that like we do.

Reply:
they want us to use this https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis but that only exists in R18.